content/pages/sprint/2023/03-hsinchu.rst

```diff
@@ -0,0 +1,77 @@
+========================================
+Scisprint 2023 March at Hsinchu
```

Comment:
It should be aligned with the title in `index.rst` : `Scisprint 2023 March in Hsinchu`